cocos2d/particle/CCParticleSystem.js

```diff
@@ -105,10 +105,11 @@ var PositionType = cc.Enum({
 
     /**
      * !#en
-     * Living particles are attached to the world but will follow the emitter repositioning.<br/>
-     * Use case: Attach an emitter to an sprite, and you want that the emitter follows the sprite.
+     * In the relative mode, the particle will move with the parent node, but not with the node where the particle is. 
+     * For example, in a moving train, the coffee in the cup lifts the fog, and the cup moves, the whole fog will not move with the cup, but from the point of view of the train, the whole fog will move with the train.
```

Comment:
the coffee in the cup is steaming. Then the steam moves (forward) with the train, rather than moves with the cup.

Reply:
Your English is awesome.